src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -345,7 +349,7 @@ void SettingsLoader::FindFragmentsAndMergeIntoUserSettings()
 void SettingsLoader::MergeFragmentIntoUserSettings(const winrt::hstring& source, const std::string_view& content)
 {
     ParsedSettings fragmentSettings;
-    _parseFragment(source, content, fragmentSettings);
+    _parseFragment(source, content, fragmentSettings, FragmentScope::User, hstring{ L"filename.json" }, true);

```

Comment:
what is this dummy filename now?

Reply:
and why are we constructing an entire hstring _to then pass as a `wstring_view`_?